recipes/_configure_server.rb

```diff
@@ -5,6 +5,9 @@
 #
 # Configures the system server.conf file
 
+default_pass4symmkey = 'changeme'
+default_ssl_password = 'password'
```

Comment:
Future, but admittedly, you use them only in one place, so you could inline them.